extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -44,7 +44,9 @@ A **feature** is a conceptual object in a virtual environment. Similar concepts
 
 Features are identified within a 3D asset by **Feature IDs**. A mesh primitive may specify multiple sets of feature IDs. These feature ID sets might (for example) identify features at different levels of abstraction: there may be feature IDs that identify individual buildings, and feature IDs that identify different parts of each building.
 
-Each feature ID set is defined as a set of values that are associated with the conceptual parts of the model. The definition of the feature ID set may include a `nullFeatureId`, which is a value that indicates that a certain part is not considered to be an identifiable object. The definition also includes a `featureCount` value, which is the number of unique features that are identified. 
+Each feature ID set is defined as a set of values that are associated with the conceptual parts of the model. The definition of the feature ID set may include a `nullFeatureId`, which is a value that indicates that a certain part is not considered to be an identifiable object. The definition also includes a `featureCount` value, which is the number of unique features that are identified.
+
+The feature ID set may may also include a `label`, and alphanumeric string used to identify feature ID sets across different glTF primitives.
```

Comment:
will there be any examples of this?

Reply:
I suppose there should be. We should have an example that uses `nullFeatureId` as well